Userland/Libraries/LibWeb/DOM/Event.idl

```diff
@@ -26,3 +25,9 @@ interface Event {
     undefined initEvent(DOMString type, optional boolean bubbles = false, optional boolean cancelable = false);
 
 };
+
+dictionary EventInit {
+    boolean bubbles = false;
+    boolean cancelable = false;
+    boolean composed = false;
+};
```

Comment:
Missing newline, CI will be sad (hopefully)

Reply:
looks like we don't run any linting on IDL files, (and look's like I forgot to re-enable that setting in CLion :sweat_smile:) fixed!